src/ralph/lib/transitions/models.py

```diff
@@ -238,10 +241,9 @@ def run_field_transition(
     """
     Execute all actions assigned to the selected transition.
     """
-    if not isinstance(instances, Iterable):
+    if not isinstance(instances, list):
```

Comment:
Then how it works when `instances` is e.g.: tuple (or any other iterable - is it possible that instances are something else then `list`?)